lib/Runner.js

```diff
@@ -103,6 +103,8 @@ function findTests(
   return new Promise(function(resolve, reject) {
     function finish() {
       const args = elmVersion ? ['--elm-version=' + elmVersion] : [];
+      // TODO: Have elmi-to-json only read test file paths, not all paths.
+      // args = args.concat(testFilePaths);
```

Comment:
More context on this: it would be more efficient to have `elmi-to-json` only read the `.elmi` files that will be used.

However, we couldn't determine how to do that, and didn't want to block the fix on that. Instead, we're filtering them out after the fact below.